src/index.ts

```diff
@@ -16,13 +17,14 @@ import {NgbTooltipModule} from './tooltip';
 import {NgbTypeaheadModule} from './typeahead';
 
 export {NgbPanelChangeEvent} from './accordion';
+export {NgbModal} from './modal';
```

Comment:
Shouldn't NgbModalRef and NgbModalOptions be exported too? They're the return type and the argument type of `NgbModal.open()`, which is the public API, so client code should be able to refer to these types, too, shouldn't they?